examples/00-fluent/ahmed_body_workflow.py

```diff
@@ -322,16 +331,16 @@
 session.results.surfaces.iso_surface["xmid"].field = "x-coordinate"
 session.results.surfaces.iso_surface["xmid"] = {"iso_values": [0]}
 
-graphics_session1 = pv.Graphics(session)
-contour1 = graphics_session1.Contours["contour-1"]
+graphics_session = Graphics(session)
```

Comment:
Visualization part can be updated further like:

>>> from ansys.fluent.visualization import GraphicsWindow, Contour
>>> contour1 = Contour(solver=session, field="velocity-magnitude", surfaces=["xmid"])
>>> disp1 = GraphicsWindow()
>>> disp1.add_graphics(contour1)

>>> contour2 = Contour(solver=session, surfaces=["xmid"])
>>> contour2.field.allowed_values
>>> contour2.field = "pressure-coefficient"
>>> disp2 = GraphicsWindow()
>>> disp2.add_graphics(contour2)

Or you can even club the displays together now:

>>> disp = GraphicsWindow(grid=(2, 1))
>>> disp.add_graphics(contour1, position=(0, 0))
>>> disp.add_graphics(contour2, position=(1, 0))

Reply:
Hi @prmukherj! Just want to keep those contour plot separate.  Regarding visualization part, Does it still required the ``set_config(blocking=True, set_view_on_display="isometric")`` config setting.

Reply:
yes, as of now. We'll soon get rid of it.